src/components/form/FormComponent.vue

```diff
@@ -1,84 +1,100 @@
+          <div class="col">
+            <component
               :is="checkVueType(field)"
-              v-for="field in row.fields"
+              v-for="field in formRow.buttons"
               :key="field.propertyName"
               :field="field"
-              @onclick="handleButtonClicked" />
+              @onclick="handleSubmit" 
+              @button-clicked="handleSubmit"/>
+          </div>
         </div>
-        <template
-            v-for="field in row.fields"
-            v-else>
-            <component
-              :is="checkVueType(field)"
-              v-if="
-                checkVueType(field) ===
-                'FormSection'
-              "
-              :key="field.propertyName"
-              v-model="field.value"
-              :field="field" />
-            <template v-else>
-              <div class="col"
-                v-show="showField(field)"
-                :key="field.propertyName"
-                :class="
-                  expressCell(field.expression)
-                ">
-                <label
-                  v-if="
-                    !hideLabelForType(
-                      field.vueType
-                    )
-                  "
-                  :for="field.propertyName"
-                  :title="field.title"
-                  :class="{
-                    mandatory: field.mandatory,
-                  }">
-                  {{ field.title }}
-                </label>
-              </div>
-              <div class="col-xl"
+        <div class="row" v-if="formRow.fields">
+          <template v-for="field in formRow.fields">
+              <component
+                :is="checkVueType(field)"
+                v-if="
+                  checkVueType(field) ===
+                  'FormSection'
+                "
                 :key="field.propertyName"
-                :class="
-                  expressCell(field.expression)
-                ">
+                v-model="field.value"
+                :field="field"
+                @value-changed="handleFieldsChanged(field)"/>
+              <template v-else>
 
-                  <component
-                    :is="checkVueType(field)"
+                <div class="col"
+                  v-show="showField(field)"
+                  :key="field.propertyName"
+                  :class="
+                    expressCell(field.expression)
+                  ">
+                  <label
                     v-if="
-                      field.vueType === 'FormButton'
+                      !hideLabelForType(
+                        field.vueType
+                      )
+                    "
+                    :for="field.propertyName"
+                    :title="field.title"
+                    :class="{
+                      mandatory: field.mandatory,
+                    }">
+                    {{ field.title }}
+                  </label>
+                </div>
+
+                <div class="col-xl"
```

Comment:
Codacy has a fix for the issue: [Expected a linebreak before this attribute.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8560396)
```suggestion
              <div
```